parsers/US_ERCOT.py

```diff
@@ -65,6 +72,17 @@
 EXCHANGE_MAPPING = {"US-CENT-SWPP": ["dcE", "dcN"], "MX-NE": ["dcL"], "MX-NO": ["dcR"]}
 
 
+# Report type IDs
+# Wind production forecast: https://www.ercot.com/mp/data-products/data-product-details?id=NP4-732-CD
+WIND_POWER_PRODUCTION_HOURLY_AVERAGED_ACTUAL_AND_FORECASTED_VALUES_RTID = 13028
+
+# Solar production forecast: https://www.ercot.com/mp/data-products/data-product-details?id=np4-737-cd
+SOLAR_POWER_PRODUCTION_HOURLY_AVERAGED_ACTUAL_AND_FORECASTED_VALUES_RTID = 13483
+
+# Load forecast: https://www.ercot.com/mp/data-products/data-product-details?id=np3-560-cd
+LOAD_FORECAST_BY_FORECAST_ZONE = 12311
+
```

Comment:
These should probably be Enums.

And while I like clear names could we make them a tiny bit shorter?